omnigibson/simulator.py

```diff
@@ -149,6 +149,9 @@ def _launch_app():
     return app
 
 
+
+    log.setLevel(logging.DEBUG if gm.DEBUG else logging.INFO)
+
```

Comment:
What is this? This doesnt even execute. Where does it actually need to go? Check with @cremebrule